gems/pending/fs/ext4/superblock.rb

```diff
@@ -223,6 +223,14 @@ def inodeSize
       isDynamic? ? @sb['inode_size'] : INODE_SIZE
     end
 
+    def isEnabled64Bit?
+      gotBit?(@sb['incompat_flags'], ICF_64BIT)
```

Comment:
It's not that expensive, but it's more expensive than just returning the value of the instance variable. It all depends on how often it's called. In the long run, many "not that expensive" things will add up.